robosuite/wrappers/domain_randomization_wrapper.py

```diff
@@ -0,0 +1,62 @@
+"""
+This file implements a wrapper for facilitating domain randomization over
+robosuite environments.
+
+TODO: make list of arguments for DRWrapper to support
+TODO: ability to restore to unmodified model / defaults
+TODO: support action noise
+TODO: support camera rotations + multiple cameras
+TODO: support image translation
+TODO: support sensor delay / repeat / dropout
+"""
+import numpy as np
+
+from robosuite.wrappers import Wrapper
+from robosuite.utils.mjmod import TextureModder, LightingModder, MaterialModder, CameraModder
+
+
+class DomainRandomizationWrapper(Wrapper):
```

Comment:
Can we provide options to perform more "controllable", e.g., using a subset of modders and specifying the subset of randomization parameters and their ranges?

Reply:
yes - I was hoping to get your input on what the ideal input interface for the wrapper initialization would look like, given the functionality we would like to support (e.g. camera, lighting, and texture modders, and each should have their own parameters for the degree of randomization, how often it happens, and whether it is "global" vs. "local" randomization)

Reply:
adding a seed option as well could help with deterministic playback and/or benchmarking

Reply:
+1 to the seed